src/editor/deserialize.ts

```diff
@@ -29,7 +29,7 @@ const LIST_TYPES = ["UL", "OL", "LI"];
 
 // Escapes all markup in the given text
 function escape(text: string): string {
-    return text.replace(/[\\*_[\]`<]|^>/g, match => `\\${match}`);
+    return text.replace(/[\\*_[\]`<]|^>/g, match => `${match}`);
```

Comment:
This replace is now doing absolutely nothing? You're replacing `match` with itself

Reply:
Yeah, you're right. I could have done better just wanted to push my changes to get assigned. So i can continue working on it later.

Reply:
Nonetheless it seems to work without any problem even though it seems illogical, I'll probably need your help. I guess I need to figure out if any other part of the project is using this escape function if not I should probably remove it what do you think or figure out a better way to handle it?

Reply:
Well no, it regresses the reason this code was added in the first place.
See https://github.com/vector-im/element-web/issues/10722 + https://github.com/matrix-org/matrix-react-sdk/pull/8004